src/Service.GraphQLBuilder/GraphQLNaming.cs

```diff
@@ -28,6 +28,11 @@ public static class GraphQLNaming
         /// <seealso cref="https://spec.graphql.org/October2021/#sec-Names.Reserved-Names"/>
         public const string INTROSPECTION_FIELD_PREFIX = "__";
 
+        /// <summary>
+        /// For any CosmosDB Query, the default alias for the container is 'c'
+        /// </summary>
+        public const string COSMOSDB_CONTAINER_DEFAULT_ALIAS = "c";
```

Comment:
I was looking for a place to put this constant. I think this place is relevant because container alias is always used while reading graphql schema.
I am open to move it to better place.